internal/nginx/manager.go

```diff
@@ -28,6 +28,9 @@ const TLSSecretFileMode = 0600
 // JWKSecretFileMode defines the default filemode for files with JWK Secrets.
 const JWKSecretFileMode = 0644
 
+// OIDCSecretFileMode defines the default filemode for files with OIDC Secrets.
+const OIDCSecretFileMode = 0644
```

Comment:
After changes in https://github.com/nginxinc/kubernetes-ingress/pull/1304/commits/a2c724f7b3e934a5be97aca6450f7fa2d5f412e7 this can be removed

```suggestion
```

Reply:
(Amazed that `make lint` didn't find this. We should probably revisit our lint config soon.)